uat/custom-components/client-java-sdk/src/main/java/com/aws/greengrass/testing/mqtt5/client/grpc/GRPCLibImpl.java

```diff
@@ -17,7 +17,7 @@ public class GRPCLibImpl implements GRPCLib {
     private final LinkFactory linkFactory;
 
     interface LinkFactory {
-        GRPCLink newLink(@NonNull String agentId, @NonNull String host, int port) throws GRPCException;
+        GRPCLink newLink(@NonNull String agentId, @NonNull String[] hosts, int port) throws GRPCException;
```

Comment:
use `List<String>` instead of `String[]` if possible. Dealing with lists is much simpler than builtin arrays.

Reply:
Will replace.

Reply:
Because PR already merged changes made in separate branch GGMQ-84-multi-homed-control-is-not-supported.ADDON

Reply:
See new PR https://github.com/aws-greengrass/aws-greengrass-client-device-auth/pull/263